lib/ReactViews/FeatureInfo/FeatureInfoSection.tsx

```diff
@@ -257,6 +259,10 @@ export class FeatureInfoSection extends React.Component<FeatureInfoProps> {
       );
     }
 
+    // Add activeStyle property
+    if (TableMixin.isMixedInto(this.props.catalogItem))
+      terria.activeStyle = this.props.catalogItem.activeStyle;
```

Comment:
In the future we might want to expose other attributes of the `activeStyle` like  the name of the table column this style maps to. So I think it might be better to define `terria.activeStyle` as an object which for now has this shape `{id: this.props.catalogItem.activeStyle}`. 

To see why this might be needed, [check this link](http://ci.terria.io/issue-6758/#share=s-aRONIRxouqIi0YaH7i1URCbVQx9&https://gist.githubusercontent.com/mwu2018/9dc97c35b163ae5af0fc7b84c9806d41/raw/d6c13107236b9fd181b09cc8e8a352be95ad6cdf/testFeatureInfoSection.json) where I have created a new "User Style" which maps to a column called 'sta_name`. In the feature info, I might sometimes want access this underlying column instead of the `id`. Making `activeStyle` an object makes it possible to expose the color column as `{id: "User Style", pointColorColumn: "sta_name"}`

Reply:
@na9da I've revised the code based on your comment.